README.md

```diff
@@ -16,6 +16,11 @@ curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poet
 (Invoke-WebRequest -Uri https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py -UseBasicParsing).Content | python
 ```
 
+### Pytest installation (Powershell)
+```powershell
+pip install pytest
```

Comment:
You've specified pytest as a poetry dependency, so instead you can rely on the user having run `poetry install`, and they can then run the tests with `poetry run pytest`

Reply:
README updated.